### PR TITLE
Add dialog for deletion in playlist.

### LINKED ISSUE #7845: Confirm Video deletion from local playlists
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

Confirming video deletion whenever a video is deleted from local playlists.

#### Why do you want this feature?

There might be a few videos whose thumbnail, link, and title are stored in the local playlist, but the video is not available. Accidental deletion can cause them to be lost forever with no way of revival. In fact thats why i made this feature request (lost a historic vid's link thats deleted)

### PR DESCRIPTION
**Pull request by samyberkane04**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Add dialog for deleting video in playlist.

#### Fixes the following issue(s)
Fix #7845

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java [hunk 1]
```diff
@@ -596,13 +596,18 @@ private void updateThumbnailUrl() {
...
+        new AlertDialog.Builder(getContext())
+                .setTitle(R.string.confirm_video_deletion_from_local_playlists)
+                .setNegativeButton(R.string.cancel, null)
+                .setPositiveButton(R.string.ok,
+                        (dialog, which) -> itemListAdapter.removeItem(item))
+                .create()
+                .show();
 
-        itemListAdapter.removeItem(item);
         if (playlistManager.getPlaylistThumbnail(playlistId)
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java:611]: All of the actions below should also be done only if the user really wants the item to be removed

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -103,6 +103,7 @@
     <string name="volume_gesture_control_summary">Use gestures to control player volume</string>
     <string name="brightness_gesture_control_title">Brightness gesture control</string>
     <string name="brightness_gesture_control_summary">Use gestures to control player brightness</string>
+    <string name="confirm_video_deletion_from_local_playlists">Are you sure to delete ?</string>
```
Comment (Stypox) [inline app/src/main/res/values/strings.xml:106]: Keep style with other similar questions throughout newpipe: 
```suggestion
    <string name="confirm_video_deletion_from_local_playlists">Do you want to remove this item?</string>
```

**Review by Stypox — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java [hunk 3]
```diff
@@ -596,13 +596,18 @@ private void updateThumbnailUrl() {
 
         changeThumbnailUrl(newThumbnailUrl);
     }
-
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java:599]: Readd this line

**Review by Stypox — CHANGES_REQUESTED** [review]

**Comment by litetex** [comment]
I would also favor @Redirion proposal.

**Pull request closed without merging**